src/store/modules/memberships.ts

```diff
@@ -63,6 +63,17 @@ class MembershipModule extends VuexModule {
         return r.data;
     }
 
+    @Action({ rawError: true })
+    async delete(id: string) {
+        await axios({
+            method: 'DELETE',
+            url: `/memberships/${id}`,
+        });
+
+        this.context.commit('unset', { id });
+        return this._all;
```

Comment:
@GarfDev Why return this? It's also not conventional to not access class properties via context. Better remove it I guess, since it also has no use:)